client/utils/calculatePosition.js

```diff
@@ -1,23 +1,42 @@
-export default function ({ newPosition, items, isFirstChild, insert = false }) {
+import InsertLocation from './InsertLocation';
+
+const { ADD_AFTER, ADD_BEFORE, REORDER } = InsertLocation;
+
+/**
+ * Calculates item position based on the options provided.
+ * @param {number} newPosition The position of the anchor item by which the new
+ * position will be calculated.
+ * @param {Object[]} items An array of objects in which we are determining
+ * item's new position.
+ * @param {boolean} isFirstChild Boolean value denoting wether the item should
```

Comment:
Should we just remove it then and use ADD_INTO instead?

Reply:
`ADD_INTO` and `isFirstChild` are contextually different. For example: action can be `REORDER` and `isFirstChild` can be `true` in case the item is reordered and is placed in the first position. Same example goes for `ADD_BEFORE` action and `isFirstChild===true`, this means item is added before other item on the first position. For that reason we _need_ two arguments.

Reply:
- Makes sense 👍 My impression because of the comment parent-child context was that it is used only for that scenario. 
- Misspelling of wether
- Please reformat param comments in google jsdoc style (https://google.github.io/styleguide/javascriptguide.xml?showone=Comments#Comments)

Thanks!